velox/common/file/FileSystems.cpp

```diff
@@ -100,6 +101,23 @@ class LocalFileSystem : public FileSystem {
     }
   }
 
+  bool exist(std::string_view path) override {
+    auto file =
+        path.find(kFileScheme) == 0 ? path.substr(kFileScheme.length()) : path;
+    return std::filesystem::exists(std::string(file));
+  }
+
+  virtual std::vector<std::string> list(std::string_view path) override {
+    auto folderPath =
+        path.find(kFileScheme) == 0 ? path.substr(kFileScheme.length()) : path;
+    const std::filesystem::path folder{folderPath};
+    std::vector<std::string> result;
+    for (auto const& dirEntry : std::filesystem::directory_iterator{folder}) {
```

Comment:
The documentation for list method says "Returns the list of files in a path.", but this code seems to return a list of files and folders in the specified folder, no?